src/javascript/JContent/actions/openInRepositoryExplorerAction.jsx

```diff
@@ -1,17 +1,20 @@
 import React from 'react';
-import {useNodeChecks} from '@jahia/data-helper';
+import {useNodeChecks, useNodeInfo} from '@jahia/data-helper';
 import PropTypes from 'prop-types';
 
 export const OpenInRepositoryExplorerActionComponent = ({path, render: Render, loading: Loading, ...others}) => {
     const res = useNodeChecks({path}, {
-        showOnNodeTypes: ['nt:base'],
-        requiredPermission: 'repositoryExplorer'
+        showOnNodeTypes: ['nt:base']
     });
-    if (res.loading && Loading) {
+    const rootRes = useNodeInfo({path: '/'}, {
```

Comment:
Agreed. I think `useNodeChecks` uses `useNodeInfo` under the hood so it might be possible to pass this as a param option. 

Params can maybe be similar to this and should behave similarly: https://github.com/Jahia/jahia-repository-explorer/blob/master/src/javascript/RepositoryExplorer/RepositoryExplorer.main.routes.jsx#L11